packages/hero/package.json

```diff
@@ -7,7 +7,7 @@
     "Jeff Escalante"
   ],
   "dependencies": {
-    "@hashicorp/js-utils": "next",
+    "@hashicorp/js-utils": "^1.0.10",
```

Comment:
This was grabbing a pre-release rather than resolving to `1.0.10` as every other `react-component` that uses this (now outdated) package does, so updated for clarity (and cause I had to touch `hero` anyways for the `g-grid-container` replacement as above).